Resources/Prototypes/Reagents/gases.yml

```diff
@@ -0,0 +1,123 @@
+﻿- type: reagent
+  id: Oxygen
+  name: oxygen
+  desc: An oxidizing, colorless gas.
+  physicalDesc: gaseous
+  color: "#808080"
+  boilingPoint: -183.0
+  meltingPoint: -218.4
+  metabolisms:
+    Gas:
+      effects:
+      - !type:Oxygenate
+        conditions:
+        - !type:OrganType
+          type: Human
+      - !type:Oxygenate
+        conditions:
+        - !type:OrganType
+          type: Animal
+      # Convert Oxygen into CO2.
+      - !type:ModifyLungGas
+        ratios:
+          CarbonDioxide: 1.0
+      - !type:ModifyLungGas
+        ratios:
+          Oxygen: -1.0
```

Comment:
Actually these two can be combined into a single ratio list, right?

Reply:
oh yeah it can